### DISCUSSION TITLE
I cannot install Seurat package

### DISCUSSION THREAD

**Discussion by alexandresilvachaves** [satijalab/seurat#4238] [Q&A] [answered]
Hello!
Good afternoon!

I'm having issues trying to install Seurat.
Recently I started using ubuntu 20.04 as my OS, however, I cannot have Seurat package installed.
I patsed just right below, the output I received from rstudio when the attempt to install the package is finished:

Error: object ‘markvario’ is not exported by 'namespace:spatstat'
Execution halted
ERROR: lazy loading failed for package ‘Seurat’
* removing ‘/home/alexandre/R/x86_64-pc-linux-gnu-library/4.0/Seurat’
Error: Failed to install 'Seurat' from GitHub:
  (converted from warning) installation of package ‘/tmp/Rtmp2naPiK/file265c577528ec/Seurat_3.9.9.9038.tar.gz’ had non-zero exit status

Any ideas of what is going on? I'd appreciate some help on this matter.

Cheers!

**Comment by pwolf-okami** [comment, marked as answer]
"object 'markaviro' is not exportet by 'namespace:spatstat'"
Has been discussed here and there- since a spatstat release from 13-3-2021 appears to be faulty you'll have to uninstall that package - and reinstall the older version; untill this is fixed I advice not to update all packages when using BiocManager as well (incase you get prompted while installing other packages f.e. as this will also update spatstat to faulty new release)

Good luck

Sources: https://github.com/mojaveazure/seurat-disk/issues/56
https://github.com/satijalab/seurat/issues/4226
